### ISSUE TITLE
Camera warping back to starting point

### ISSUE THREAD

**Issue by Lithdel** [imchillin/Anamnesis#485]
I havent figured out how to replicate this because it seems to only do it in certain zones, My first thought was that it had to do with negative values in the camera position but that does not seem to be the case as the bug occurs in Shirogane but not at Quarrymill (Both locations have a negative camera location but only Shirogane causes issues), Hullbreaker Isle is another location this glitch happens in

Basically in the affected zones, If you unlock the camera and move it, the camera will constantly warp from where you moved it, back to the starting point, and then back to where you moved it again.
